packages/warehouses/package.json

```diff
@@ -10,26 +10,26 @@
     "dependencies": {
         "@google-cloud/bigquery": "^5.9.1",
         "@lightdash/common": "^0.291.1",
-        "snowflake-sdk": "^1.6.4",
+        "@lightdash/databricks-sql": "^0.1.8-beta.3",
```

Comment:
should we keep beta version ?

Reply:
@rephus as the original package is with beta label - I think yes. we can keep it that way until they release a new version and then we can switch back to their original package.